### ISSUE TITLE
spreadsheet merge cells

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#2695]
Merge & Centre function on the fly (something I thought of after your presentation)

Frequently ledgers will have multiple accounts on a single page that would benefit from header-like formatting in the middle of the page

you mentioned that the spreadsheet in the transcription works with the same functions as any other spreadsheet so I am wondering about the possibility of a "merge and centre" function executed on the fly

For example, on this page: 

it is record for both William Howard Platt AND John W. Lucas 

It would be great if we could do a "merge and centre" mid-page for John W. Lucas to delineate the new/different account information.

**Comment by saracarl** [comment]
https://collections.digitalmaryland.org/digital/collection/p17340coll20/id/18818/rec/2

**Comment by saracarl** [comment]
Lydia at SSM

**Comment by saracarl** [comment]
This points to a tension between "reproducing what you see on the page exactly as you see it on the page" and "producing usable data".  Your suggestions leans strongly to the first, but someone who wanted to understand who made which transactions for how much on what dates would prefer the account owner to be listed -- or at least derivable -- from a spreadsheet export as a separate field for each transaction.  Ben is working on a similar ledger, but he's using markdown tables instead of spreadsheets.  Each account holder is identified as a section header, and when he exports that header is available as part of the spreadsheet sheet for the following table.  See http://fromthepage.com/benwbrum/jeremiah-white-graves-diaries/jeremiahwhitegravesdiaryvolume1part3/display/572190
In other words, it's a tradeoff and we're leaning towards the second for the spreadsheet functionality.  (Of course, if others ask for the same thing, we could definitely change our mind, and I could also see letting project owners decide for themselves, so this isn't a "no" as much as a "not right now")

**Comment by benwbrum** [comment]
This request has not recurred in the last year or so, and given how unclear it is to address I'm closing it.
